test/test_data/test_validation.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+
+from autosklearn.data.validation import InputValidator
+
+import numpy as np
+import pandas as pd
+
+
+class InputValidatorTest(unittest.TestCase):
+
+    def setUp(self):
+        self.X = [
+            [2.5, 3.3, 2, 5, 1, 1],
+            [1.0, 0.7, 1, 5, 1, 0],
+            [1.3, 0.8, 1, 4, 1, 1]
+        ]
+        self.y = [0, 1, 0]
+
+    def test_array_input(self):
+        validator = InputValidator()
+        X, y = validator.validate(self.X, self.y)
+
+        self.assertTrue(isinstance(X, np.ndarray))
+        self.assertTrue(isinstance(y, np.ndarray))
+
+    def test_numpy_input(self):
```

Comment:
Could you please add a test for scipy.sparse, too?

Reply:
added in commit!